re.sonny.Workbench.Devel.json

```diff
@@ -5,11 +5,21 @@
   "sdk": "org.gnome.Sdk",
   "sdk-extensions": [
     "org.freedesktop.Sdk.Extension.vala",
+    "org.freedesktop.Sdk.Extension.rust-stable",
     "org.gnome.Sdk.Docs"
   ],
   "build-options": {
-    "append-path": "/usr/lib/sdk/vala/bin",
-    "append-ld-library-path": "/usr/lib/sdk/vala/lib"
+    "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm14/bin:/usr/lib/sdk/vala/bin",
+    "append-ld-library-path": "/usr/lib/sdk/vala/lib",
+    "build-args": ["--share=network"],
```

Comment:
just in case, this will need to be removed because flathub ci doesn't allow network for builds

Reply:
Although direct uploads is a thing now so if it's really too annoying, we might look into using that.

Reply:
We only need that for Devel.
Will remove it from the other manifest.